client/src/main.js

```diff
@@ -13,10 +13,11 @@ import { faAnglesLeft } from "@fortawesome/free-solid-svg-icons";
 library.add(faAnglesUp, faAnglesRight, faAnglesLeft);
 
 import BaseHeader from "@components/BaseHeader.vue";
-
+import NavbarAuth from "@components/NavBarAuth.vue";
 Vue.config.productionTip = false;
 Vue.component("font-awesome-icon", FontAwesomeIcon);
 Vue.component("base-header", BaseHeader);
+Vue.component("navbar-auth", NavbarAuth);
```

Comment:
to sm šele zdj vidu.. a to če kle definiraš komponento, pol je available povsod in ne rabiš importat v komponenti kjer jo hočeš uporabit? 

če je tko, se men to ne zdi najboljša ideja, da so kr "global" oz na globalni vue instanci..

Reply:
Ja, to mam že od začetka to `base-header` componento, zdej sm dodal še navbar-auth. Točn tko, mišljeno je da je na globalni ravni, in da jo vsaka komponenta lahko uporabi. 
Bom dal vse kr na local.